lib/react.js

```diff
@@ -250,6 +250,12 @@ declare module react {
   declare export var ConcurrentMode: ({children: ?React$Node}) => React$Node; // 16.7+
   declare export var StrictMode: ({children: ?React$Node}) => React$Node;
```

Comment:
(Should these also be using React$ComponentType? You can't call them as a function.)

Reply:
Good catch. I'm ok with adding that fix to this PR or doing it in a separate one.

Reply:
@bvaughn actually brought up a good point below, it's not quite a ComponentType. It _will_ be an AbstractComponent, though!

Reply:
Let's leave it for now?

Reply:
Yeah, sounds good